rfc/2019-12-11-Talent-Plan-Courses.md

```diff
@@ -0,0 +1,35 @@
+# Proposal: Talent Plan Courses
+
+- Author(s): [Ling Jin](https://github.com/JinLingChristoher)
+- Last updated: 2019-12-11
+- Discussion at: [Incubating Program: Talent Plan Courses](https://github.com/pingcap/community/issues/130)
+
+## Abstract
+
+Since there are plenty of people interested in TiDB but lack background knowledge, so we propose to create a series of computer system courses to help those people enter the field of the TiDB ecosystem, also computer system infrastructure. Course participants should have developed good knowledge and skills in system programming after consumed course materials.
```

Comment:
```suggestion
Since there are plenty of people interested in TiDB but lack background knowledge, so we propose to create a series of courses to help those people enter the field of infrastructure system and TiDB ecosystem. Course participants should have developed good knowledge and skills in system programming after consumed course materials.
```